whosnext.py

```diff
@@ -70,6 +70,8 @@
     '2022-02-15': ['Jan Heinen'],
     '2022-03-01': ['Rado Dukalski'],
     '2022-03-15': ['Marco Reijne'],
+    '2022-03-29': ['Leila Alizadehsaravi'],
+    '2022-04-12': ['Francesca Andretta'],
```

Comment:
Might as well add Jeswin too, so we can run it Tuesday.

Reply:
I didn't commit his presentation yet in case he would get sick or wouldn't be able to present otherwise, but I added him before running the whosnext today.